### PR TITLE
STD-13 | Entity type configurations for EF Core

### PR DESCRIPTION
**Pull request by zefirlover**

### CODE REVIEW

File: StudySharp.Domain/Models/Learning/Course.cs [hunk 1]
```diff
@@ -12,16 +12,14 @@ public class Course : IWithDateCreated, IWithDateModified
 
         //connection part
         public List<Tag> Tags { get; set; }
-
```
Comment (SleepyF0X) [inline StudySharp.Domain/Models/Learning/Course.cs:15]: Why you remove TeacherId?

File: StudySharp.Domain/Models/Learning/PracticalBlock.cs [hunk 2]
```diff
@@ -5,7 +5,10 @@ public class PracticalBlock
         public int Id { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
+        
+        //connection part
         public int QuizId { get; set; }
         public Quiz Quiz { get; set; }
+        public Course Course { get; set; }
```
Comment (SleepyF0X) [inline StudySharp.Domain/Models/Learning/PracticalBlock.cs:12]: Where CourseId?

File: StudySharp.Domain/Models/Learning/TheoryBlock.cs [hunk 3]
```diff
@@ -5,5 +5,8 @@ public class TheoryBlock
         public int Id { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
+        
+        //connection part
+        public Course Course { get; set; }
```
Comment (SleepyF0X) [inline StudySharp.Domain/Models/Learning/TheoryBlock.cs:10]: Where CourseId?

File: StudySharp.DomainServices/Configurations/CourseConfiguration.cs [hunk 4]
```diff
@@ -22,10 +22,23 @@ public void Configure(EntityTypeBuilder<Course> builder)
             builder
                 .Property(_ => _.DateCreated)
                 .IsRequired();
+            
+            // connections
```
Comment (SleepyF0X) [inline StudySharp.DomainServices/Configurations/CourseConfiguration.cs]: Please dont do this, its obviously
Delete comments like this in all models

File: StudySharp.DomainServices/Configurations/CourseConfiguration.cs [hunk 5]
```diff
@@ -22,10 +22,23 @@ public void Configure(EntityTypeBuilder<Course> builder)
...
+
+            builder
+                .HasMany(_ => _.TheoryBlocks)
+                .WithOne(_ => _.Course);
 
-            //TODO: update this functional in scope STD-13
             builder
-                .Ignore(_ => _.Content);
+                .HasMany(_ => _.PracticalBlocks)
+                .WithOne(_ => _.Course);
```
Comment (SleepyF0X) [inline StudySharp.DomainServices/Configurations/CourseConfiguration.cs]: You must configure it with method HasForeignKey()
Or something like this

**Review by WerewolfNet — APPROVED** [review]

**Pull request merged**